vitup/src/config/initials/snapshot.rs

```diff
@@ -0,0 +1,103 @@
+use crate::config::initials::Role;
+use chain_crypto::PublicKeyFromStrError;
+use chain_impl_mockchain::value::Value;
+use hersir::builder::Wallet as WalletSettings;
+use jormungandr_lib::crypto::account::Identifier;
+use serde::{Deserialize, Serialize};
+use thor::{Wallet, WalletAlias};
+use voting_hir::VoterHIR;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Initials {
+    pub tag: String,
+    pub content: Vec<Initial>,
+}
+
+impl Default for Initials {
+    fn default() -> Self {
+        Self {
+            tag: "".to_string(),
+            content: Vec::new(),
+        }
+    }
+}
+
+impl Initials {
+    pub fn as_voters_hirs(
+        &self,
+        defined_wallets: Vec<(&WalletAlias, &WalletSettings)>,
+    ) -> Result<Vec<VoterHIR>, Error> {
+        let mut voter_hirs = Vec::new();
+
+        for initial in self.content.iter() {
+            match initial {
+                Initial::Random { count, level, role } => {
+                    for _ in 0..*count {
+                        voter_hirs.push(VoterHIR {
+                            voting_key: Wallet::default().account_id(),
```

Comment:
doesn't this always produce the same key?

Reply:
i think not, because it is (should be) randomi:

https://github.com/input-output-hk/jormungandr/blob/master/testing/thor/src/wallet/mod.rs#L79